src/make_features/subject_verb_object/README.MD

```diff
@@ -0,0 +1,12 @@
+### Subject Verb Object Autocomplete
+
+These scripts are part of the process of extracting subject verb object relationships between words in content item titles.
+
+## Generating new data for SVO Autocomplete
+
+1. Download the latest copy of the preprocessed content store from the `govuk-data-infrastructure-integration` [S3 bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-data-infrastructure-integration?region=eu-west-1&prefix=knowledge-graph)
```

Comment:
https://s3.console.aws.amazon.com/s3/buckets/govuk-data-infrastructure-integration?region=eu-west-1&prefix=knowledge-graph/&showversions=false seems to work better